power-platform/admin/geo-to-geo-migrations.md

```diff
@@ -28,131 +29,146 @@ The Geo Migration feature allows customers to move their environments in a singl
 > - After making a request, expect at least 10 days for the migration to be completed.  
 > - Geo migrations are not supported into or out of US GCC, US GCC High, or China.
 > - Geo migrations are restricted into or out of OCE or IND.
-> - The Dynamics 365 Marketing app does not support geo migration, due to component dependencies. For more information, see [Manage your Dynamics 365 Marketing instances](/dynamics365/marketing/manage-marketing-instances). If installed, the Dynamics 365 Marketing app must be uninstalled prior to the migration.
-> - Geo migration is not supported for Microsoft Dataverse for Teams environments.
-> - Dataverse organization linked to a finance and operations organization is not supported.
```

Comment:
Should we document that FnO is supported as long as both are moved? And it requires unlinking?

Reply:
I think we can leave out internal details such as moving FnO and unlinking as long as there's no further impact to the customer (other than the expected downtime).